ArduCopter/ArduCopter.cpp

```diff
@@ -118,6 +118,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
     SCHED_TASK(gcs_send_heartbeat,     1,    110),
     SCHED_TASK(gcs_send_deferred,     50,    550),
     SCHED_TASK(gcs_data_stream_send,  50,    550),
+    SCHED_TASK(update_telemetry,     400,     50),
```

Comment:
Indeed it seems to be little excessive, I think that about ~10Hz should be enough to refresh the log sending and the reading of command.I am inquiring about this problem. Also the usage of the task scheduler for telemetry action might change in the next version.